happi/cli.py

```diff
@@ -255,7 +261,9 @@ def happi_cli(args):
         logger.debug('Starting load block')
         logger.info(f'Creating shell with devices {args.device_names}')
         devices = {}
-        for name in args.device_names:
+        names = " ".join(args.device_names)
+        names = names.split()
```

Comment:
I'm happy to merge this (just got back from vacation), quick question: is this so that the whitespace-delimited set of names isn't interpreted as a single string in your load example?

Reply:
@ZLLentz thanks for your review. You are exactly correct.

To be honest I think I was being a bit dumb when I made the PR. I have no need for the extra quotes in my command. `happi load $(happi search "*" --names)` works just great without the changes you highlighted.

On the other hand it might be nice default behavior for any strings given as arguments to be whitespace separated before interpretation as item names.

I'm happy keeping or removing the changes in lines 264 265. Just let me know what you prefer.

Reply:
I think, even though it's a bit confusing and not strictly necessary for the stated purpose of the PR, it's a good improvement. We force the names to be valid Python identifiers in `HappiItem`, which means we can't have any names with whitespace in them anyway. Others are bound to run into the same thing.

I'll merge this as-is, thank you for the contribution!